src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -4473,3 +4473,152 @@ def merge():
         2  bar      7
         3  bar      8
         """
+
+    def to_csv():
```

Comment:
Maybe dumb question; but why does this not have parameters in the signature?

Reply:
This is a dummy dataframe just to override docstrings.